lib/linguist/heuristics.yml

```diff
@@ -169,6 +182,15 @@ disambiguations:
     pattern: '^\s*(?:%%|main\s*\(.*?\)\s*->)'
   - language: JavaScript
     pattern: '(?m:\/\/|("|'')use strict\1|export\s+default\s|\/\*.*?\*\/)'
+- extensions: ['.ex']
+  rules:
+  - language: Elixir
+    pattern:
+    - '^\s*@moduledoc\s+'
+    - '^\s*(cond|import|quote|unless)\s+'
+    - '^\s*def(exception|impl|macro|module|protocol)(\(|\s+)'
```

Comment:
This can be simplified into:

~~~suggestion
    - '^\s*def(?:exception|impl|macro|module|protocol)[(\s]'
~~~